q_yaar_platform/account/models.py

```diff
@@ -54,6 +54,18 @@ class PlatformUser(AbstractBaseUser, AbstractExternalFacing, AbstractTimeStamped
     def __str__(self):  # pragma: no cover
         return self.email
 
+    class Meta:
+        indexes = [models.Index(fields=["email"]), models.Index(fields=["external_id"])]
```

Comment:
Indexing is super important for efficient querying. However, it is important to carefully consider what to index and what not to index. Indexing is also expensive (extremely space heavy).